src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey(items: [], key: string): GroupsMap<Object> {
+  const group: GroupsMap<Object> = {};
+
+  items.forEach((item) => {
+    if (item[key]) {
```

Comment:
The check if 'item[key]' exists could potentially be problematic if the value is 0 or any other falsy value. Try to be more specific with your checks.